core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -1185,6 +1186,15 @@ public SqlValidatorScope getOverScope(SqlNode node) {
     return scopes.get(node);
   }
 
+  @Override public boolean shouldValidateInCreateProcedure(SqlNode node) {
+    return (node instanceof SqlScriptingNode
```

Comment:
These extra parentheses around the return statement appear to be causing the linter check to fail.